electron/main.ts

```diff
@@ -39,7 +39,7 @@ app
   })
 
 app.once('window-all-closed', () => {
-  cleanUpAndQuit()
+  if (process.platform !== 'darwin') app.quit()
```

Comment:
This would lead to an issue where app subprocesses, such as Nitro, model unload, API server, are not being killed properly.
```suggestion
  if (process.platform !== 'darwin') cleanUpAndQuit()
```